scalableminds/webknossos
# Webglperf

Johannes and I refactored the WebGL handling of skeleton tracings in the frontend. It should be able to handle more many more nodes now. 

Mailable description of changes (needs to be understandable by webknossos mailing list people):
- Improved the performance on the rendering of SkeletonTracings in the frontend
- wk frontend should be able handle thousands of skeleton nodes (we tried up to 1mio nodes successfully)


Steps to test:
- Trace in Skeleton Mode
  - Test all 3 skeleton modes
  - shuffle colors
  - create / delete nodes
  - merge and split trees
  - create / delete trees
- Test Volume mode (should be unaffected by these changes, but let's play it safe)

Issues:
- fixes #1711 

------
